type/impl/src/main/java/iu/type/ComponentFactory.java

```diff
@@ -86,41 +85,26 @@ static void checkIfAlreadyProvided(ComponentArchive alreadyProvidedArchive, Comp
 	/**
 	 * Creates a modular component.
 	 * 
-	 * @param parent             parent component
 	 * @param parentLayer        {@link ModuleLayer} to extend
+	 * @param parent             parent component
 	 * @param parentLoader       {@link ClassLoader} for parent delegation
```

Comment:
It looks like the order should be parent, parentLoader, parentLayer in the Javadoc.